__tests__/app.test.js

```diff
@@ -209,7 +210,7 @@ describe("POST /api/reviews/:review_id/comments", () => {
       .send(newComment)
       .expect(404)
       .then(({ body }) => {
-        expect(body.msg).toEqual("Not Found");
+        expect(body.msg).toEqual("Review ID does not exist");

```

Comment:
The `.toBe` matcher would be preferable here as only primitives are being compared